### PR TITLE
HappyThoughts API

### PR DESCRIPTION
**Pull request by petrabrinck**
Really great project to work with even though I had some problems that Jennie helped me with :-)

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -7,23 +7,82 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
 app.use(cors())
 app.use(bodyParser.json())
-
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+
+// app.get('/', (req, res) => {
+//   res.send('Hello world hej hej')
+// })
```
Comment (JennieDalgren) [inline server.js]: remove if not used :)

File: server.js [hunk 2]
```diff
@@ -7,23 +7,82 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+  res.json(thoughts)
+})
+
+app.post('/', async (req, res) => {
+  //Retrieve the information sent by the client to our API endpoint
+  // const { message, hearts } = req.body.message
+  const { message } = req.body
+  // Use our mongoose model to create the database entry
+  // const thought = new Thought({ message, hearts })
+  const thought = new Thought({ message })
```
Comment (JennieDalgren) [inline server.js]: good with comments, but cleanup unused code

File: server.js [hunk 3]
```diff
@@ -7,23 +7,82 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+    // Success
+    const savedThought = await thought.save()
+    res.status(201).json(savedThought)
+  } catch (err) {
+    res.status(400).json({ message: 'Could not save your heart message', error: err.errors })
+  }
+})
+
+// The endpoint updates the number of like
+app.post('/:thoughtId/like', async (req, res) => {
```
Comment (JennieDalgren) [inline server.js]: ⭐️

**Comment by petrabrinck** [comment]
Tack 😊🙏🏽 Ska städa upp!

//Petra

**Pull request closed without merging**